app/javascript/src/components/Dashboard/Contacts/constants.jsx

```diff
@@ -9,14 +9,21 @@ export const CONTACTS_FORM_INITIAL_FORM_VALUES = {
   first_name: "",
   last_name: "",
   email: "",
-  role: "",
+  role: null,
 };
 
 export const CONTACTS_FORM_VALIDATION_SCHEMA = yup.object().shape({
   first_name: yup.string().required("First Name is required"),
   last_name: yup.string().required("Last Name is required"),
   email: yup.string().email("Invalid email").required("Email is required"),
-  role: yup.string().required("Role is required"),
+  role: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string(),
+      value: yup.string(),
```

Comment:
Refer the comments in Slack thread. There was a suggestion regarding using `oneOf`.

Reply:
I have resolved this using and used oneOf to validate label and value